Libs/Widgets/ctkCheckablePushButton.h

```diff
@@ -65,12 +55,33 @@ class CTK_WIDGETS_EXPORT ctkCheckablePushButton : public ctkPushButton
   void setIndicatorAlignment(Qt::Alignment indicatorAlignment);
   Qt::Alignment indicatorAlignment()const;
 
+  /// Get checked state of the checkbox on the button.
   virtual Qt::CheckState checkState()const;
+  /// Set checked state of the checkbox on the button.
   virtual void setCheckState(Qt::CheckState checkState);
 
+  /// By default the checkbox is connected to the checkable property of the push button.
+  /// You can change this behaviour by clearing the "checkBoxControlsButton"
+  /// property.
+  /// \note In checkBoxControlsButton mode, calling setCheckable instead of
```

Comment:
I think we need to change `setCheckable` to `setCheckable()` so that doxygen properly auto link with the corresponding documentation. Could you check by building the doc locally ?

Reply:
Changed to setCheckable(). Doxygen links correctly to the functions now.